app/lib/email_parameter_filter_proc.rb

```diff
@@ -0,0 +1,12 @@
+class EmailParameterFilterProc
+  # email regexp from regular-expressions.info/email.html
+  EMAIL_REGEXP = /\b[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}\b/i
```

Comment:
Optional, but you could use `URI::MailTo::EMAIL_REGEXP` from Ruby's standard library, which is equivalent to the more longwinded:
`/\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z/`

This would mean not having to maintain this RegEx, plus would align with how we're validating form fillers' email addresses in forms-runner.
```suggestion
  EMAIL_REGEXP = URI::MailTo::EMAIL_REGEXP
```

Reply:
Oooh, I was wondering if there was something in the standard library for that! Your google-fu is obviously better than mine 😂 I couldn't find this documented anywhere so didn't realise this exists.

Reply:
I'll make that change.

Reply:
Ah, the URI::MailTo regexp has anchors that expects the whole string to be an email address 🤔

Reply:
Ah, OK. Nothing wrong with the way you wewre doing it originally 😁